MekHQ/src/mekhq/campaign/mission/AtBDynamicScenarioFactory.java

```diff
@@ -2263,7 +2270,7 @@ public static int getAeroLanceSize(int unitTypeCode, boolean isPlanetOwner, Stri
      * @param entityList
      */
     private static void deployArtilleryOffBoard(List<Entity> entityList) {
-        OffBoardDirection direction = OffBoardDirection.getDirection(Compute.randomInt(4) + 1);
+        OffBoardDirection direction = OffBoardDirection.getDirection(Compute.randomInt(4));
```

Comment:
Is this a bug fix?

Reply:
Yeah, I kept getting "invalid off board direction crashes" because 4 is not a valid off board direction (it's 0-3).

Which I guess answers my earlier question if "does randomInt()" include the upper bound number.